### PR TITLE
sci-biology/ants: install scripts

### PR DESCRIPTION
**Pull request by TheChymera**
ANTs comes with a umber of scripts which call its core functions - and which are what many researchers preferentially use.

### CODE REVIEW

File: sci-biology/ants/ants-9999.ebuild [hunk 1]
```diff
@@ -22,6 +22,9 @@ src_install() {
 	cd "${WORKDIR}/${P}_build/ANTS-build" || die "build dir not found"
 	default
 	cd "${WORKDIR}/${P}/Scripts" || die "scripts dir not found"
+	exeinto /usr/bin
+	doexe *.sh
```
Comment (jlec) [inline sci-biology/ants/ants-9999.ebuild]: This is what ```dobin```does.

File: sci-biology/ants/files/99ants [hunk 2]
```diff
@@ -0,0 +1 @@
+ANTSPATH=/usr/bin/
```
Comment (jlec) [inline sci-biology/ants/files/99ants]: Is this really needed?

Comment (TheChymera) [inline sci-biology/ants/files/99ants]: yes, sadly it is. Some ANTs scripts check if the variable is set when they are called and fail otherwise.

**Comment by TheChymera** [comment]
@jlec ?

**Pull request merged**